### PR TITLE
Feature/pxweb2 149 common text styling

### PR DESCRIPTION
**Pull request by PerIngeVaaje**
Make use of text-styles in common scss

### CODE REVIEW

**Review by MikaelNordberg — APPROVED** [review]
Looks good

File: libs/pxweb2-ui/src/lib/components/Typography/Label/Label.tsx [hunk 1]
```diff
@@ -1,31 +1,33 @@
 import cl from 'clsx';
 import classes from './Label.module.scss';
```
Comment (SjurSutterudSagen) [inline libs/pxweb2-ui/src/lib/components/Typography/Label/Label.tsx:2]: Maybe add a blank line between imports from packages and imports from within the project? To better see which imports are what? The same applies to Button.tsx and Tag.tsx too.

```suggestion

import classes from './Label.module.scss';
```

**Review by SjurSutterudSagen — DISMISSED** [review]
Looks good to me! We probably want more feedback on the placement of the text-styles.scss file next to the global.scss file. But it makes sense to me.

**Review by SjurSutterudSagen — APPROVED** [review]
LGTM

**Pull request merged**